pkg/agent/flowexporter/exporter/exporter_perf_test.go

```diff
@@ -0,0 +1,258 @@
+// +build !race
+
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import (
+	"crypto/rand"
+	"flag"
+	"fmt"
+	"math/big"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/vmware/go-ipfix/pkg/registry"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/agent/flowexporter"
+	"antrea.io/antrea/pkg/agent/flowexporter/connections"
+	"antrea.io/antrea/pkg/agent/flowexporter/flowrecords"
+)
+
+const (
+	testNumOfConns           = 20000
+	testNumOfDenyConns       = 20000
+	testNumOfDyingConns      = 2000
+	testNumOfInactiveRecords = 2000
+	testNumOfIdleDenyConns   = 2000
+	testBufferSize           = 1048
+)
+
+var recordsReceived = 0
+
+/*
+Sample output:
+	go test -test.v -run=BenchmarkExport -test.benchmem -bench=BenchmarkExportConntrackConns -memprofile memprofile.out -cpuprofile profile.out
+	goos: linux
+	goarch: amd64
+	pkg: antrea.io/antrea/pkg/agent/flowexporter/exporter
+	cpu: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
+	BenchmarkExportConntrackConns (truncated output)
+		exporter_perf_test.go:79:
+			Summary:
+			Number of conntrack connections: 20000
+			Number of dying conntrack connections: 2000
+			Total connections received: 18703
+	BenchmarkExportConntrackConns-2   	      75	  13750074 ns/op	  965550 B/op	   22268 allocs/op
+	PASS
+	ok  	antrea.io/antrea/pkg/agent/flowexporter/exporter	5.494s
+*/
```

Comment:
Add the output that you have for the varying number of connections. It will be good to show that.

Reply:
Done. Thanks!